extensions/amp-carousel/0.1/scrollable-carousel.js

```diff
@@ -61,6 +61,7 @@ export class AmpScrollableCarousel extends BaseCarousel {
       'white-space': 'nowrap',
       'overflow-x': 'auto',
       'overflow-y': 'hidden',
+      'touch-action': 'pan-x',
```

Comment:
There seems to be no css for scrollable carousel - everything is being set here. just following existing patterns